packages/playground/src/views/sshkey_view.vue

```diff
@@ -115,258 +100,196 @@
-      createCustomToast(`The activation of ${key.name} key has been enabled.`, ToastType.success);
-      key.activating = false;
-    };
-
-    const setInactiveKey = async (key: SSHKeyData) => {
-      key.activating = true;
-      key.isActive = false;
-      await updateSSHKeysInChain();
-      createCustomToast(`The activation of ${key.name} key has been disabled.`, ToastType.success);
-      key.activating = false;
-    };
-
-    const deleteKey = async (selectedKeys: SSHKeyData[]) => {
-      deleting.value = true;
-      const ids: number[] = selectedKeys.map(key => key.id);
-      allKeys.value = allKeys.value.filter(_key => !ids.includes(_key.id));
-      selectedKeys.map(key => (key.deleting = true));
-      await updateSSHKeysInChain();
-      selectedKeys.map(key => (key.deleting = false));
-      createCustomToast(
-        `The selected ${selectedKeys.length > 1 ? "keys" : "key"} has been deleted successfully.`,
-        ToastType.success,
-      );
-      deleting.value = false;
-    };
-
-    const generateSSHKeys = async (key: SSHKeyData) => {
-      generatingSSH.value = true;
-      const keys = await generateKeyPair({
-        alg: "RSASSA-PKCS1-v1_5",
-        hash: "SHA-256",
-        name: key.name,
-        size: 4096,
-      });
-
-      generatedSSHKey.value = keys.publicKey;
-      key.fingerPrint = calculateFingerprint(keys.publicKey);
-      downloadAsFile("id_rsa", keys.privateKey);
-      createCustomToast(`${key.name} key has been generated successfully.`, ToastType.success);
-      generatingSSH.value = false;
-    };
-
-    const exportKeys = async (keys: SSHKeyData[] | number[]) => {
-      isExporting.value = true;
-      let exportKeys: SSHKeyData[] = [];
-
-      if (Array.isArray(keys) && keys.length > 0 && typeof keys[0] === "number") {
-        exportKeys = allKeys.value.filter(key => keys.includes(key.id as SSHKeyData & number));
-      } else {
-        exportKeys = keys as SSHKeyData[];
-      }
+const openDialog = (type: SSHCreationMethod) => {
+  dialogType.value = type;
+};
+
+const closeDialog = () => {
+  dialogType.value = SSHCreationMethod.None;
+};
+
+const exportAllKeys = () => {
+  isExporting.value = true;
+  sshKeysManagement.export(allKeys.value);
+  isExporting.value = false;
+};
+
+const exportSelectedKeys = (keys: SSHKeyData[]) => {
+  isExporting.value = true;
+  let exportKeys: SSHKeyData[] = [];
+
+  if (Array.isArray(keys) && keys.length > 0 && typeof keys[0] === "number") {
+    exportKeys = allKeys.value.filter(key => keys.includes(key.id as SSHKeyData & number));
+  } else {
+    exportKeys = keys as SSHKeyData[];
+  }
+
+  exportKeys.forEach(key => {
+    delete key.deleting;
+    delete key.activating;
+  });
+
+  sshKeysManagement.export(exportKeys);
+  isExporting.value = false;
+};
+
+const updateActivation = async (key: SSHKeyData) => {
+  activating.value = key.activating = true;
+
+  try {
+    await sshKeysManagement.update(allKeys.value);
+  } catch (e: any) {
```

Comment:
in this catch, we should look up for "insufficientBalanceError" and create a toast with friendly error message, 
and the else block should handle other cases with something like " something went wrong"

Reply:
The ssh script handle this.